src/main/java/core/basesyntax/dao/impl/AbstractDao.java

```diff
@@ -1,11 +1,54 @@
 package core.basesyntax.dao.impl;
 
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
 
-public abstract class AbstractDao {
+public abstract class AbstractDao<T> {
     protected final SessionFactory factory;
 
     protected AbstractDao(SessionFactory sessionFactory) {
         this.factory = sessionFactory;
     }
+
+    public T create(T entity) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Content entity", e);
```

Comment:
it seems that this message is suitable only for comment :)

let's change the messages to more general, i.e. "Can't insert/remove entity"

and since the test is asking for a specific message for comment - override `create()` in CommentDaoImpl

Reply:
Why does this suit only for comment? This is a message that is required by test